src/components/organisms/NewOrderBookWidget/NewOrderBookWidget.stories.tsx

```diff
@@ -180,3 +180,8 @@ Loading.args = {
 Loading.parameters = {
     ...RESPONSIVE_PARAMETERS,
 };
+
+export const Itayose = Template.bind({});
```

Comment:
Added Itayose variant to NewOrderBookWidget, using the InfoTooltip present in OrderBookWidget. Ben hasn't provided a design yet but in the interest of delivering this on Monday, I am going ahead with this